src/util.ts

```diff
@@ -81,15 +85,16 @@ export function getPropertyMap(
       member.type === 'PropertyDefinition' &&
       member.static &&
       member.key.type === 'Identifier' &&
-      member.key.name === 'properties' &&
-      member.value?.type === 'ObjectExpression'
+      member.key.name === 'properties'
     ) {
-      for (const prop of member.value.properties) {
+      const arg = member.value as ESTree.ObjectExpression;
+      for (const prop of arg.properties) {
         if (prop.type === 'Property') {
-          const name = getIdentifierName(prop.key);
+          const key = prop.key as ESTree.Identifier;
```

Comment:
similarly this may not be an identifier. you should just pass the uncasted `prop.key` to `getIdentifierName` and probably change the property entry interface to have `key: Node` rather than `key: Identifier`